### PR TITLE
sc-controller: update to 0.4.4.

### PR DESCRIPTION
**Pull request by Vintodrimmer**

### CODE REVIEW

**Comment by cr6git** [comment]
Please take into account the changes in https://github.com/void-linux/void-packages/pull/1717

**Comment by Vintodrimmer** [comment]
Wow you are fast.
Changed template to comply with yours.

File: srcpkgs/sc-controller/template [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # Template file for 'sc-controller'
```
Comment (maxice8) [inline srcpkgs/sc-controller/template]: extra line

Comment (Vintodrimmer) [reply srcpkgs/sc-controller/template]: fixed

**Pull request merged**